### DISCUSSION TITLE
Errors after update to 1.3

### DISCUSSION THREAD

**Discussion by technosprout** [thuliteio/doks#1184] [Q&A]
ERROR render of "page" failed: "/Users/dev/site/node_modules/@hyas/doks-core/layouts/_default/baseof.html:4:5": execute of template failed: template: articles/single.html:4:5: executing "articles/single.html" at <partial "head/body-class" .>: error calling partial: partial "head/body-class" not found
Built in 88 ms
Error: error building site: render: failed to render pages: render of "page" failed: "/Users/dev/site/node_modules/@hyas/doks-core/layouts/_default/baseof.html:4:5": execute of template failed: template: legal/single.html:4:5: executing "legal/single.html" at <partial "head/body-class" .>: error calling partial: partial "head/body-class" not found

**Comment by h-enk** [comment]
> partial "head/body-class" not found

Check your [mounts](https://getdoks.org/migration-guides/v-1/v-1-3/#update-mounts) — make sure it has

```toml
## layouts
[[mounts]]
  source = "node_modules/@hyas/core/layouts"
  target = "layouts"
```

**Reply by technosprout** [reply]
These lines exist in module.toml
however the directory "node_modules/@hyas/core/layouts" does not

**Reply by h-enk** [reply]
Do you have a repository you could share?

**Reply by technosprout** [reply]
I ended up recreating the repo and moved my content over. no ideal but worked
